src/osgearth.mk

```diff
@@ -0,0 +1,27 @@
+# This file is part of MXE. See LICENSE.md for licensing information.
+
+PKG             := osgearth
+$(PKG)_WEBSITE  := http://osgearth.org/
+$(PKG)_DESCR    := Geospatial SDK for OpenSceneGraph
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 2.9
+$(PKG)_CHECKSUM := 22aeef42bb700c1e669d9ba57ce4155c0668caf86cc738750a6b9d34c1eaf2a4
+$(PKG)_GH_CONF  := gwaldron/osgearth/tags, osgearth-
+$(PKG)_DEPS     := curl gcc gdal openscenegraph sqlite tinyxml2 zlib
```

Comment:
It seems to actually use tinyxml and the build fails for me on shared targets. Can you make a gist of one of the shared logs?

Reply:
There's not much in the log but I see what's likely happened - you started from clean (good practice) and didn't have tinyxml present so it used its internal one. I'd just use the internal one as our package will be removed eventually.

Reply:
Also, use `cc` instead of `gcc` an place it first in deps. `gcc` is only really for people what want to build a barebones compiler.

Reply:
> There's not much in the log but I see what's likely happened - you started from clean (good practice) and didn't have tinyxml present

Yes, that is the case.

> so it used its internal one

I'll check it.

> Also, use `cc` instead of `gcc` an place it first in deps.

Just forgot to update this dependency. This package was prepared in 2017 and only now I have found time to propose it to upstream...